src/Instrumentation/IO/README.md

```diff
@@ -0,0 +1,67 @@
+# OpenTelemetry IO auto-instrumentation
+
```

Comment:
maybe also point to the auto-instrumentation readme here, for other installation methods. Or, move this text into auto-instrumentation readme?

Reply:
> maybe also point to the auto-instrumentation readme here, for other installation methods. Or, move this text into auto-instrumentation readme?

@brettmc  Not sure if I understand it, when you say methods, you mean packages?
BTW. I planned to add the same section to all supported `auto` packages

Reply:
I think the documentation on how to install the extension should just be written once, in the extension's readme: https://github.com/open-telemetry/opentelemetry-php-instrumentation/blob/main/README.md#installation
Then, we link to that from all `auto` packages (which reduces some repetition). wdyt?

Reply:
This is not only about extension, `open-telemetry/opentelemetry-instrumentation-installer` installs everything auto-instrumentation need, but yes we can add description there and link to that.

Reply:
@brettmc Is it better now?

Reply:
Maybe call out that the easiest way to install auto-instrumentation _plus implementations_ is the installer... or manually via <link to extension's readme> ?
I just want to make sure all options are documented, eg the installer might not be the best fit for automation/dockerfile etc

Reply:
I agree that we should document all options. Will update it